src/material-experimental/mdc-chips/public-api.ts

```diff
@@ -18,3 +18,8 @@ export * from './chip-default-options';
 export * from './chip-icons';
 export * from './chip-text-control';
 export * from './chip-edit-input';
+
+export {
+  MatChipListChange,
+  MatChipList,
```

Comment:
I'm not totally sure about these two, because the MDC chips doesn't have an equivalent.

Reply:
Yeah I don't think we should re-export these. Maybe just create a text file of exceptions

Reply:
The reason I did it is that the migration won't be as straightforward anymore, similar to https://github.com/angular/components/pull/20960#discussion_r518390435

Reply:
I think we're planning to do some work on the migration script to try automate moving people to the correct MDC component a little more